src/openrct2/drawing/Drawing.Sprite.cpp

```diff
@@ -361,6 +362,36 @@ bool gfx_load_csg()
     }
 }
 
+std::optional<rct_gx> GxfLoadGx(const std::vector<uint8_t>& buffer)
+{
+    try
+    {
+        OpenRCT2::MemoryStream istream(buffer.data(), buffer.size());
+        rct_gx gx;
+
+        gx.header = istream.ReadValue<rct_g1_header>();
+
+        // Read element headers
+        gx.elements.resize(gx.header.num_entries);
+        read_and_convert_gxdat(&istream, gx.header.num_entries, false, gx.elements.data());
+
+        // Read element data
+        gx.data = istream.ReadArray<uint8_t>(gx.header.total_size);
+
+        // Fix entry data offsets
+        for (uint32_t i = 0; i < gx.header.num_entries; i++)
+        {
+            gx.elements[i].offset += reinterpret_cast<uintptr_t>(gx.data.get());
+        }
+        return gx;
```

Comment:
```suggestion
        return std::optional(gx);
```
a does not compile on linux, b does not compile on windows

Reply:
you can just do `{ gx }` but tbh it shouldcompile fine wirth `return gx`

Reply:
windows does not like `{ gx }` either: 'no instance of constructor "std::optional<_Ty>::optional [with _Ty=rct_gx]" matches the arguments list
argument types are: (rct_gx)

Reply:
`std::make_optional(std::move(gx));`